datasets/Van_de_Schoot_PTSD/convert_clean_schoot_lgmm_ptsd.ipynb

```diff
@@ -0,0 +1,282 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import pandas as pd\n",
+    "from risparser import convert_ris_to_csv\n",
+    "os.chdir(\"/automated-systematic-review-datasets/datasets/Van_de_Schoot_PTSD\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Convert RIS files into CSV"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Input file: raw/schoot-lgmm-ptsd-initial.ris\n",
+      "Number of articles: 6187\n",
+      "Export file: csv/schoot-lgmm-ptsd-initial.csv\n",
+      "Input file: raw/schoot-lgmm-ptsd-included-1.ris\n",
+      "Number of articles: 363\n",
+      "Export file: csv/schoot-lgmm-ptsd-included-1.csv\n",
+      "Input file: raw/schoot-lgmm-ptsd-included-3.ris\n",
+      "Number of articles: 8\n",
+      "Export file: csv/schoot-lgmm-ptsd-included-3.csv\n",
+      "Input file: raw/schoot-lgmm-ptsd-included-2.ris\n",
+      "Number of articles: 38\n",
+      "Export file: csv/schoot-lgmm-ptsd-included-2.csv\n"
+     ]
+    }
+   ],
+   "source": [
+    "raw_files = [raw_files for raw_files in os.listdir(\"raw\") if raw_files.endswith(\".ris\")]\n",
```

Comment:
Since last week these .ris files are also available on the open science framework, https://osf.io/4hqk6/. 
It would be nice if this notebook downloads the scripts directly from there!

Reply:
Oh please ignore this comment, the location of these `.ris` files might change @Rensvandeschoot?